pages/whats-new/2022-01-28-v08.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Grouparoo v0.8 adds a Model-centric UI and more!"
+date: "2022-01-28"
+tags: ["new"]
+description: "Grouparoo v0.8 adds the ability to integrate with Airtable, Rollbar, Klaviyo, and Vero!"
+---
+
+The 0.8 release had a big focus on user experience with the addition of our Model-Centric UI!
+
+<Image
+  alt="Salesforce Destinations"
+  src="v08-release/model-centric-ui.png"
+  width={1000}
+  height={424}
+/>
+
+Additionally we have added a few more integrations:
+
+- Airtable
+- Rollbar
```

Comment:
Rollbar isn't a source/destination, but rather a monitoring tool - maybe it shouldn't be grouped in with these?

Reply:
What are we doing with it then?

Reply:
We have a few of these types of plugins (Sentry, Rollbar, New Relic...) that don't provide sources or destinations, but are used for sending metrics / reporting errors from the Grouparoo app itself to these various services. It's kind of its own category of plugins.